src/components/SummaryNameMapModal.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+    Select,
+    Button,
+    ModalContent,
+    ModalHeader,
+    ModalCloseButton,
+    ModalBody,
+    ModalFooter,
+    FormControl,
+    FormLabel,
+    ModalOverlay,
+    VStack
+} from '@chakra-ui/react';
+import React, { useEffect, useState } from 'react';
+import trpc from "../trpc";
+import ModalWithBackdrop from 'components/ModalWithBackdrop';
+import ReactMarkdown from 'react-markdown';
+import { MinUser } from 'shared/User';
+import { SummaryNameMap } from 'shared/Summary';
+
+export function SummaryNameMapModal({ nameMap, groupUsers, onClose }: {
+    nameMap: SummaryNameMap | undefined
```

Comment:
when nameMap is expected to be undefined? document it:

```
/**
 * @param nameMap ...
 */
```

Reply:
a warning is added instead when nameMap is undefined